### DISCUSSION TITLE
Relation between segments and job

### DISCUSSION THREAD

**Discussion by g-kartik** [cvat-ai/cvat#3426] [Ideas]
There is a one to many relationship between segment and job. Means a segment can contain multiple job. This is not really the case, I think. A segment relates to just one job.
So in Job table instead of Foreign Key relationship, it should be a One to One relationship to Segment table.

**Comment by nmanovic** [comment]
@g-kartik , your observation is correct. But we are going to have multiple jobs per segment in the future.
